### ISSUE TITLE
Unable to hit entitys with no mods enabled

### ISSUE THREAD

**Issue by elliott867158** [fr1kin/ForgeHax#197]
So i was using boatfly and auto water bucket to go to my witch farm afk spot, but now i can't seem to break my boat. I can hit blocks just fine, but entitys don't work. I'm using the 1.15.2 version btw

**Comment by fr1kin** [comment]
Unbind any keys bound to a mouse button (Autoclicker select is by default bound to middle click). Save then restart the game.

**Comment by elliott867158** [comment]
so you mean having the r key bound to select block is a problem?

**Comment by fr1kin** [comment]
No, its only an issue with mouse buttons. Something in Minecrafts code does not like it when other settings use mouse buttons for binds. You gotta unbind any setting bound to a mouse key (in the Controls menu) then restart because the stupid code causing this issue runs only at startup.

**Comment by elliott867158** [comment]
Thanks, but i've decided not to use forgehax anymore, all i needed was bookbot
